python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -87,15 +88,41 @@ def __init__(
         self._metadata = metadata
 
     @property
-    def num_nodes(self) -> int:
-        """Returns the number of nodes in the graph.
-
+    def num_nodes(self):
+        """
```

Comment:
We may need to include more comprehensive comments, possibly with code examples, to demonstrate the usage of `num_nodes`. Please refer to the docstring for `num_nodes` in DGL, available at: https://github.com/dmlc/dgl/blob/master/python/dgl/heterograph.py#L2491-L2528, for an illustrative example.